src/password_to_curve.rs

```diff
@@ -0,0 +1,23 @@
+use crate::group::Group;
+use curve25519_dalek::{edwards::EdwardsPoint, ristretto::RistrettoPoint};
+
+use generic_array::GenericArray;
+use hkdf::Hkdf;
+
+pub trait PasswordToCurve: Group {
```

Comment:
How about calling this MapToCurve instead of PasswordToCurve, at least?

And then how about doing:

pub trait GroupWithMapToCurve: Group { ... } ?